frontend/components/product-list/sections/FilterableProductsSection/FacetsAccordion.tsx

```diff
@@ -98,7 +98,24 @@ export const FacetAccordionItem = forwardRef<FacetAccordionItemProps, 'div'>(
       const isDisabled = isProductListEmpty || !hasApplicableRefinements;
 
       const formattedFacetName = formatFacetName(attribute);
-      const isHidden = !hasApplicableRefinements && !isProductListEmpty;
+      let isHidden = !hasApplicableRefinements && !isProductListEmpty;
```

Comment:
To keep this boolean immutable, I personally prefer

```suggestion
      const isHidden = !hasApplicableRefinements && !isProductListEmpty && !isWorksin && !isToolsCategoryOnPartsPage;
```

or something like that

Reply:
Resolved 451fdd85a5550cd34a1f611c1383149be57d57ac